### DISCUSSION TITLE
Root programming forum?

### DISCUSSION THREAD

**Discussion by PaulBappoo** [iRobotEducation/create3_docs#458] [Questions]
### How are you connecting to your ~Create 3~ Root robot?

Bluetooth (Python SDK)

### Computer(s) Model(s) and Operating System(s)

Macbook pro 2019

### Describe your question.

Hi, I'm starting to code python for the Root RT1 and looking for a forum where I can chat with other users, but cant find a Root forum anywhere, is this something I can discuss here, or does anyone know of anywhere I could chat with appropriate people?

**Comment by shamlian** [comment]
Hi! Thanks for the note. At the moment, there isn't a specific forum to chat with other [iRobot Education Python SDK](https://github.com/iRobotEducation/irobot-edu-python-sdk/issues) or Root robot users, though I have seen conversations over the usual social media suspects. I think we want to keep this forum for Create 3 robots, but if you have an issue you could certainly drop one into the irobot-edu-python-sdk repo, and likewise we will look into some solution for building a community space for Root robots (and their users 😉) in the future.
